### DISCUSSION TITLE
Scope class description

### DISCUSSION THREAD

**Discussion by rolandgvc** [google/flax#502] [General] [answered]
Shouldn't the Scope class description be a bit more "descriptive"? Currently it is defined just as:
```python
class Scope:
  """Scope."""
```

**Comment by avital** [comment, marked as answer]
I think @jheek is working on improving docstrings. Jonathan, is there a PR or branch @rolandgvc can take a look at in the meanwhile?

**Reply by jheek** [reply]
It's the `Scope` class itself that has this placeholder the methods themselves have actual docstrings.
That said core still lakes a more high-level documentation of the abstraction.
For now I would like at the design tests (tests/core/design) to learn more about functional core
